### PR TITLE
Feature/31/mobile menu

### PR DESCRIPTION
**Pull request by azinkamran**
#31

### CODE REVIEW

**Review by bytewiz — CHANGES_REQUESTED** [review]
Looks good 👍 but can we change all the class names to use `kebab-case` and lowercased letters instead? :) 

Example this `MobileMenu-menuwrap` will be this `mobile-menu-menuwrap`

File: src/client/components/MobileMenu/MobileMenu.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
+.MobileMenu-menuwrap {
```
Comment (Fatemeh-Pakpour) [inline src/client/components/MobileMenu/MobileMenu.css]: Good job, I have a few comment. The mobile menu does not look good in some of the mobile versions like GalaxyS5, iphone 6/7/8. I sent the screen shot of them on the slack to see what I mean.

File: src/client/components/MobileMenu/MobileMenu.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+.MobileMenu-toggler:checked + .MobileMenu-hamburgur > div {
+  transform: rotate(135deg);
+}
+/* Turn line to X */
+.MobileMenu-toggler:checked + .MobileMenu-hamburgur > div::before,
+.MobileMenu-toggler:checked + .MobileMenu-hamburgur > div::after {
+  top: 0;
+  transform: rotate(90deg);
+}
+.MobileMenu-toggler:checked:hover + .MobileMenu-hamburgur > div {
```
Comment (Fatemeh-Pakpour) [inline src/client/components/MobileMenu/MobileMenu.css]: The hamburger menu should place on the right of the page. Just follow the design on this link :
https://www.figma.com/file/QYeFmfAU7XAaewnskogsxH/Class10-Final-Project-Mockups?node-id=0%3A1

**Review by Fatemeh-Pakpour — CHANGES_REQUESTED** [review]

File: src/client/components/MobileMenu/MobileMenu.css (hunk 1 above)
Comment (azinkamran) [reply src/client/components/MobileMenu/MobileMenu.css]: thanks for the feedback :), I resolved it

File: src/client/components/MobileMenu/MobileMenu.css (hunk 2 above)
Comment (azinkamran) [reply src/client/components/MobileMenu/MobileMenu.css]: thanks for the feedback:)

**Pull request merged**